server/builder/receiver.js

```diff
@@ -1,45 +1,55 @@
 const Queue = require('../rabbitmq')
 const execFile = require('child_process').execFile
-// const fs = require('fs')
-// const config = require('../src/config')
+const config = require('../src/config')
+const rp = require('request-promise')
+const fs = require('fs')
 
 receiver().then(console.log('receiver started'))
 
 async function receiver () {
   const queue = new Queue('build-raven-os-org')
 
-  await queue.receive((msg) => {
+  await queue.receive(async (msg) => {
     const data = [...msg.content]
-    console.log('message received', data)
+    const url = config.url + 'manifest/'
+    const manifests = []
+    const path = '../../build-raven-os-org/server/manifests/manifest_'
```

Comment:
Hmm you're going one path up too much, nah? Therefore assuming the project's directory is `build-raven-os-org` (which it may not be)?

A double check would be nice!